docs/developer/advanced/sharing-saved-objects.asciidoc

```diff
@@ -334,12 +339,56 @@ image::images/sharing-saved-objects-step-5.png["Sharing Saved Objects ESO migrat
 
 NOTE: Reminder, don't forget to add unit tests and integration tests!
 
+[[sharing-saved-objects-phase-2]]
+=== Phase 2 Developer Flowchart
```

Comment:
You might already be planning for this, but do you think it would be worthwhile to add a note/warning indicating that all steps within `Phase 2` need to be done in the same release?

Reply:
Good point, I'll change the flowchart accordingly